src/exchange/exchange.service.js

```diff
@@ -196,6 +197,21 @@ angular
                 });
         }
 
+        setConfiguration (organization, serviceName, data) {
+            return this.services.APIExchange.put("/{organizationName}/service/{exchangeService}", {
```

Comment:
We should begin using ovh-api-services imo :)

https://github.com/ovh-ux/ovh-api-services